test/unit/node/app.test.ts

```diff
@@ -107,18 +107,6 @@ describe("createApp", () => {
     app.dispose()
   })
 
-  it("should change the file mode of a socket", async () => {
```

Comment:
Flakey test so decided we should remove it. Happy to add back if desired.

Reply:
It seems odd to me that this flakes so it could be an actual bug.  IMO we leave tests even if flaky.

Reply:
Good call. I'll add back in.